requirements.in

```diff
@@ -0,0 +1,3 @@
+-e .[async,cli,pandas,sqlalchemy]
+pandas<1.0.0
```

Comment:
why dup pandas here? isn't it included in `.[pandas]`?

Reply:
@mistercrunch let me look into it. The constraint should be in `setup.py` if possible (assuming that `pip-compile` can handle this). I know that extra requirements are challenging for `pip-compile` (hence the reason for the addition of the `requirements.in` file).